content/ja/handbook/legal/customer-negotiations.md

```
@@ -0,0 +1,272 @@
+
+注意: このハンドブックページ内の「GitLab 法務」または「法務」への言及はすべて、コマーシャル法務チームを指します。
+
+(1) [業務的](#operational) - ディールをクローズするための要件、法務運営に関する一般的な FAQ（財務やディールデスクに関する並行点を含む）を含む、GitLab 法務との連携および関与方法。
+
+(2) [教育的](#educational) - 法務について学び、顧客が法務に関連する質問を持っている場合により自信を持って対応できるよう積極的なアドバイスや教育を含む、販売サイクル中の営業をより良くサポートするための契約の仕組み。
+
+このページに含まれる情報は、次のように業務的と教育的の2つの区分に整理されています：
+
+| 業務的 | 教育的 |
+| ------ | ------ |
+| [**法務コマーシャルチームへの連絡方法**](#how-to-reach-the-legal-commercial-team) | [**GitLab 契約の概要**](#overview-of-gitlab-agreements)  |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)